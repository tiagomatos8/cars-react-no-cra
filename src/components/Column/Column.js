import React from 'react'
import PropTypes from 'prop-types';

/**
 * Column - A responsive column
 * https://bulma.io/documentation/columns/basics/
 * 
 * @param {string} size Define the size of each column individually https://bulma.io/documentation/columns/sizes/
 */
const Column = props => {
  return (
    <div className={`Column-component column ${props.size}`}>
      {props.children}
    </div>
  )
}

Column.propTypes = {
  size: PropTypes.string
}

export default Column