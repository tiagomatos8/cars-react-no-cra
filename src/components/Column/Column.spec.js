import React from 'react';
import renderer from 'react-test-renderer';
import Column from './Column';

describe('Column Component Test Suite', () => {

  it('Render Column component with no props', () => {
    const component = renderer.create(<Column />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});