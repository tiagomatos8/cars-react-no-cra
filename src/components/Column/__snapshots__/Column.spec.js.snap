// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column Component Test Suite Render Column component with no props 1`] = `
<div
  className="Column-component column undefined"
/>
`;
